Project Walking/Source code/app/src/main/java/com/teamapricot/projectwalking/Reminder.java

```diff
@@ -0,0 +1,34 @@
+package com.teamapricot.projectwalking;
+
+import android.content.Context;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+public class Reminder {
+    private Context ctx;
+
+    public Reminder(Context mctx) {
+        this.ctx=mctx;
+    }
+
+
+    //building the notification text and drop down arrow
+    public void addNotification( String notificationTitle, String notificationMessage,String channel_id, int notificationRequestCode) {

```

Comment:
Add JavaDoc comment to the method. Include @param descriptions for each parameter.